docs/usage.rst

```diff
@@ -219,6 +219,33 @@ Reporting on non-accounting data (queries)
 ------------------------------------------
 Check the module `mis_builder_demo <https://github.com/OCA/mis-builder/tree/10.0/mis_builder_demo>`_ to see how to create specific series of data for Committed purchases not yet invoiced.
```

Comment:
This explanation about committed purchases is not really related to the Queries tab. It's probably better to mention it in a section that talks about selecting "data sources" when configuring a column.

Reply:
This can be done in another PR of course.

Reply:
in another PR!